spotipy/client.py

```diff
@@ -1562,26 +1562,16 @@ def _get_uri(self, type, id):
         return "spotify:" + type + ":" + self._get_id(type, id)
 
     def _search_multiple_markets(self, q, limit, offset, type, markets, total):
-        results = {
-                type + 's': {
-                    'href': [],
-                    'items': [],
-                    'limit': limit,
-                    'next': None,
-                    'offset': 0,
-                    'previous': None,
-                    'total': 0
-                }
-            }
+        results = {}
         for country in markets:
             result = self._get(
                 "search", q=q, limit=limit, offset=offset, type=type, market=country
             )
-            results[type + 's']['href'].append(result[type + 's']['href'])
-            results[type + 's']['items'] += result[type + 's']['items']
-            results[type + 's']['total'] += result[type + 's']['total']
-            if total and len(results[type + 's']['items']) >= total:
+            results[country] = result
+            if total and len(results[country][type + 's']['items']) >= total:
```

Comment:
Let's store `type + 's'` into a variable first and then later on use this variable here and below as required. That should improve readability and should be a bit faster as it wouldn't have to concatenate the same strings again every time.